.github/workflows/schema.yaml

```diff
@@ -26,3 +26,24 @@ jobs:
         run: |
           test -f schema/rockcraft.json &&
           diff generated_schema.json schema/rockcraft.json
+
+  validate-schema:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout rockcraft
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+
+      - name: Install ajv cli
+        run: npm install -g ajv-cli
+
+      - name: Run ajv
+        run: |
+          ajv validate -s schema/rockcraft.json -d docs/reference/code/example/rockcraft.yaml --strict=false --spec=draft2020
```

Comment:
Let's make `schema/rockcraft.json` a global env var since it's used in a couple of places

Reply:
Done